.github/workflows/ci.yml

```diff
@@ -642,6 +642,43 @@ jobs:
           path: diagnostics*.gz
           retention-days: 1
 
+  unit-benchmark:
+    name: Unit Benchmarks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          profile: minimal
+
+      - name: Setup Cache for Rust and Benchmarks
+        uses: Swatinem/rust-cache@v2
+        with:
+          # Additional non workspace directories, separated by newlines
+          cache-directories: ./benches
```

Comment:
I think that we shouldn't use rust-cache to cache previous result

Reply:
`rust-cache` is for caching rust build artifacts not for other purposes

Reply:
Yeah I was trying to accomplish the benchmarks caching as well using the `cache-directories` option from `rust-cache` but given that its not working and that `rust-cache` does a great job caching rust artifacts and deps. I will be using `actions/cache` for the single purpose of caching our benchmark file as shown in the benchmarks repo.